ivanti_security_controls/plugin.spec.yaml

```
@@ -226,3 +454,110 @@ actions:
         description: Agent status with details
         type: agent_status
         required: true
+  start_patch_scan:
+    title: Start Patch Scan
+    description: Start patch scan
+    input:
+      hostnames:
+        title: Hostnames
+        description: Hostnames. Either hostnames or machine group IDs must be specified
```

```suggestion
        description: Hostnames - Either hostnames or machine group IDs must be specified
```

I'm not sure about that edit. "Hostnames" is not a sentence. Not sure how to handle that. You can dismiss if wanted.